### PR TITLE
Doc/Freemarker quick start

### PR DESCRIPTION
**Pull request by pengw00**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ Website for findeasily.net
 (*Note*: You need to purchase licenses before using these 2 resources into your personal project)
 1. The UI template I bought: [Official Demo Link with Images](http://citybook.kwst.net/)
 2. The Email template I bought: [Lil Commerce](http://notification-emails.com/transactional-email-templates) 
+3. UI: how freemarker work in Spring to make java developer happy.
```
Comment (zhouhao) [inline README.md]: 👍 
BTW: work -> works

**Comment by zhouhao** [comment]
Are you be able to merge it?
If not, please find me in slack, let’s figure it out

**Comment by pengw00** [comment]
cannot-merge

**Comment by zhouhao** [comment]
I guess you can merge now

**Pull request merged**